packages/core/src/env-set/utils.ts

```diff
@@ -0,0 +1,45 @@
+import { getEnv } from '@silverhand/essentials';
+
+class EnvParseError extends Error {}
+class InvalidValueFormatError extends Error {}
```

Comment:
These class declarations are useless if no constructor is provided here.  Could move those error message construction logic here, in you intend to define a custom Error Class.

Reply:
good call. updated.